lib/req/test.ex

```diff
@@ -154,6 +156,17 @@ defmodule Req.Test do
 
   require Logger
 
+  @typedoc """
+  A stub is an atom that scopes stubbed requests.
+
+  A common choice for this is the module name for the Req-based client that you
+  are using `Req.Test` for. For example, if you are using `MyApp.Weather` as
+  your client, you can use `MyApp.Weather` as the stub name in functions
+  like `stub/1` and `stub/2`.
+  """
+  @typedoc since: "0.4.15"
+  @type stub() :: atom()
```

Comment:
What do you think about changing this to a typep or an opaque? I believe in principle it can be any term and I'd like to reserve the right to change it to such if I ever need it.

Reply:
Yep good call, done.